workspace-automation.md

```diff
@@ -33,3 +33,21 @@ This commands taps into our AWS Data Pipeline to see its current capacity.
 
 How it looks like in Slack:
 ![/pipeline](https://user-images.githubusercontent.com/2837532/72271944-cf55fc80-35f5-11ea-971c-e75a55148e67.png)
+
+## Asana
```

Comment:
I would move this whole section to the beginning, prior to the legion slash commands

Reply:
oh this reminds me to write the github based updates and git based release automation docs too (the latter i may just link to the blog post lel)

Reply:
kk @tlowande @woozyking i revised a bit, added some new screen shots as well as adding in annotations on the screen shots...lemme know if the annotations are too overwhelming 🌚